postgres/Dockerfile

```diff
@@ -23,3 +22,4 @@ ENV PGDATABASE="postgres"
 # tasks from a Dockerfile. This workaround checks, on each bashrc eval, if the
 # PostgreSQL server is running, and if not starts it.
 RUN printf "\n# Auto-start PostgreSQL server.\n[[ \$(pg_ctl status | grep PID) ]] || pg_start > /dev/null\n" >> ~/.bashrc
+CMD ["sh", "-c", "'mkdir -p $PGDATA && initdb -D $PGDATA'", "&&", "sh"]
```

Comment:
Unfortunately, setting `CMD` won't work, because Gitpod needs it and overwrites it just before starting a workspace.

Instead, I think we should use `~/.bashrc` to `mkdir` and `initdb` on-the-fly when needed. (Or maybe `pg_start` should do it?)

Reply:
I took the liberty of fixing this myself by pushing a commit to your PR. 🙂 I hope you don't mind.